### PR TITLE
Yolo-v5 tflite model 

### PR DESCRIPTION
**Pull request by neso613**
Any pull request you open is subject to the TensorFlow Hub Terms of Service at www.tfhub.dev/terms.

### CODE REVIEW

File: assets/docs/neso613/yolo-v5-tflite/tflite_model/1.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Lite neso613/yolo-v5-tflite/tflite_model/1
+
+TF Lite quantized version of the Yolo-v5 for object detection.
+
+<!-- parent-model: neso613/yolo-v5-tflite/1 -->
+<!-- asset-path:  -->
```
Comment (maringeo) [inline assets/docs/neso613/yolo-v5-tflite/tflite_model/1.md:6]: You would need to provide the path to the tflite file. One way to do this is to upload it to some cloud storage (the tflite file needs to be publicly readable).

**Review by WGierke — CHANGES_REQUESTED** [review]
Hi @neso613,
Just like in PR #58, we also need you to host the TFLite file on a world-readable remote location for this PR. Feel free to reach out if you have any questions!

**Comment by WGierke** [comment]
Thank you for your contribution @neso613 ! It'll take a few hours until the model is published. Once that happened, you can  remove the uploaded TFLite file again from your Cloud bucket if you want.

**Comment by neso613** [comment]
Thanks @WGierke.

**Comment by neso613** [comment]
@WGierke I have checked tfhub.dev. Model is not published yet. How much time it will take?

**Comment by neso613** [comment]
@maringeo Does the model publishing took time?

**Comment by maringeo** [comment]
Hi @neso613, our publishing pipeline experienced some issues unrelated to this PR. The problem is close to being resolved, so I expect the model to be published soon (in the next day or two).

**Comment by WGierke** [comment]
The model is live now: https://tfhub.dev/neso613/lite-model/yolo-v5-tflite/tflite_model/1 Sorry for the delay! The missing architecture tag should be added within a few days.

**Comment by neso613** [comment]
Thanks for updating me!

**Pull request merged**